src/components/content/order/formDataHelpers/deployParamsHelper.ts

```diff
@@ -13,7 +18,8 @@ export const getDeployParams = (
     selectServiceHostingType: UserOrderableServiceVo.serviceHostingType,
     selectArea: string,
     selectRegion: string,
-    selectFlavor: string
+    selectFlavor: string,
+    selectContactServiceDetails: ServiceProviderContactDetails | undefined
```

Comment:
just thinking if we can make remove undefined because contact details are mandatory in the OCL.